### PR TITLE
meld: update to 3.22.0.

### PR DESCRIPTION
**Pull request by icp1994**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

**Comment by icp1994** [comment]
I don't think I can do something about that failure.

File: srcpkgs/meld/template [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 distfiles="${GNOME_SITE}/meld/${version%.*}/meld-${version}.tar.xz"
-checksum=f48e10eec606f687a87061e78668f6bb40e63e032175c4c7033636b65a157d13
+checksum=3fc107c98ef6e75358ffd2b0d14c85ddb48fe14a11e939a94322faaa8e90c40d
 
-post_install() {
-	rm $DESTDIR/usr/share/glib-2.0/schemas/gschemas.compiled \
-		$DESTDIR/usr/share/icons/hicolor/icon-theme.cache
+do_check() {
+	ninja -C build test
+	python3 -m pytest test
```
Comment (classabbyamp) [inline srcpkgs/meld/template:21]:
```suggestion
post_check() {
	python3 -m pytest test
```
does this work? the default `do_check()` should work, probably

Comment (icp1994) [reply srcpkgs/meld/template]: yep, that works fine

**Comment by icp1994** [comment]
https://github.com/void-linux/void-packages/commit/a3e43925f092a03b3ca917b2c8c7039fccd9ee9f oh well...

**Pull request closed without merging**